replication/replication.proto

```diff
@@ -254,6 +254,9 @@ message GetVolumeReplicationInfoResponse {
   // This field is OPTIONAL.
   // The value of this field MUST NOT be negative.
   int64 last_sync_bytes = 3;
+  // Holds the latest replication state.
+  // This field is REQUIRED.
+  string replication_state = 4;
```

Comment:
Just curious, as we will have predictable states, can this not be an enum instead?

Reply:
> How are you going to use this value in csi-addons then?

We will fetch the state and parse it in ceph-csi and pass the value accordingly to the api.
CSI-Addons doesn't need to validate anything then, and just update the `Status.State` value based on the api's response.

> its marked as required too so it may break compatibility ?

Hmm, yeah we would have to mark it optional for backward compatibility.